focus-android/app/build.gradle

```diff
@@ -250,62 +250,62 @@ dependencies {
-    implementation "org.mozilla.components:feature-findinpage:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-intent:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-prompts:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-session:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-search:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-tabs:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-toolbar:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-top-sites:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-sitepermissions:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:lib-crash:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:lib-crash-sentry:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:lib-state:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:feature-media:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:lib-auth:${AndroidComponents.VERSION}"
-    implementation "org.mozilla.components:lib-publicsuffixlist:${AndroidComponents.VERSION}"
-
-    implementation "org.mozilla.components:service-glean:${AndroidComponents.VERSION}", {
+    implementation project(':browser-engine-gecko')
+    implementation project(':browser-domains')
+    implementation project(':browser-errorpages')
+    implementation project(':browser-icons')
+    implementation project(':browser-menu')
+    implementation project(':browser-state')
+    implementation project(':browser-toolbar')
+
+    implementation project(':concept-awesomebar')
+    implementation project(':concept-engine')
+    implementation project(':concept-fetch')
+    implementation project(':concept-menu')
+
+    implementation project(':compose-awesomebar')
+
+    implementation project(':feature-awesomebar')
+    implementation project(':feature-app-links')
+    implementation project(':feature-customtabs')
+    implementation project(':feature-contextmenu')
+    implementation project(':feature-downloads')
+    implementation project(':feature-findinpage')
+    implementation project(':feature-intent')
+    implementation project(':feature-prompts')
+    implementation project(':feature-session')
+    implementation project(':feature-search')
+    implementation project(':feature-tabs')
+    implementation project(':feature-toolbar')
+    implementation project(':feature-top-sites')
+    implementation project(':feature-sitepermissions')
+    implementation project(':lib-crash')
+    implementation project(':lib-crash-sentry')
+    implementation project(':lib-state')
+    implementation project(':feature-media')
+    implementation project(':lib-auth')
+    implementation project(':lib-publicsuffixlist')
```

Comment:
For the record, I added this line today because of a merge conflict. `lib-publicsuffixlist` was added recently in https://github.com/mozilla-mobile/focus-android/commit/a9a09b171665081b4e5f4d426f0e30f22f84ae49#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9R287